packages/x-date-pickers/src/DigitalClock/DigitalClock.tsx

```diff
@@ -85,6 +85,16 @@ type DigitalClockComponent = (<TDate>(
   props: DigitalClockProps<TDate> & React.RefAttributes<HTMLDivElement>,
 ) => React.JSX.Element) & { propTypes?: any };
 
+/**
+ * Demos:
+ *
+ * - [TimePicker](https://mui.com/x/react-date-pickers/time-picker/)
```

Comment:
WDYT about adding `DigitalClock`, `MultiSectionDigitalClock` and `TimeClock` to `time-picker.md` and `date-time-picker.md` to actually tie these components to more demos?
Although, it's a more broad question, about how nested we want to be with the linking on the documentation.
If we do what I propose, then we should also adjust `date-time-picker.md` to also include the relevant `DateCalendar` component... 🤷 
Any thoughs @mui/explore? 🤔

Reply:
I did add them for now, since it did make sense to me as well ... feel free to point that out and/or revert later on @mui/explore